projects/packages/my-jetpack/_inc/components/evaluation-recommendations/index.tsx

```diff
@@ -1,18 +1,61 @@
 import { Container, Col, Text } from '@automattic/jetpack-components';
-import { Flex, FlexItem, DropdownMenu } from '@wordpress/components';
+import { Icon, Flex, FlexItem, DropdownMenu, Button } from '@wordpress/components';
 import { __, _n } from '@wordpress/i18n';
 import { moreHorizontalMobile } from '@wordpress/icons';
-import { useEffect } from 'react';
+import { chevronLeft, chevronRight } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useState, useEffect, useCallback, useRef } from 'react';
 import useEvaluationRecommendations from '../../data/evaluation-recommendations/use-evaluation-recommendations';
 import useAnalytics from '../../hooks/use-analytics';
 import { JetpackModuleToProductCard } from '../product-cards-section/all';
 import styles from './style.module.scss';
 import type { FC } from 'react';
 
 const EvaluationRecommendations: FC = () => {
+	const containerRef = useRef( null );
 	const { recordEvent } = useAnalytics();
 	const { recommendedModules, redoEvaluation, removeEvaluationResult } =
 		useEvaluationRecommendations();
+	const [ isAtStart, setIsAtStart ] = useState( true );
```

Comment:
Not sure it's necessary to extract it in this PR, but I think it would be nice if all this scroll stuff was in a component called something like `<ScrollContainer />`. Not sure how easy/realistic it is to make it reusable like that but I think that would be best case scenario 🤔

Reply:
I think it's a good idea to make that reusable, but I don't think that should be the case for now, since we are not really using it anywhere else (and I can't imagine another use of it in the foreseeable future).